sway-lsp/src/core/traverse_typed_tree.rs

```diff
@@ -5,7 +5,7 @@ use crate::{
     utils::token::{struct_declaration_of_type_id, to_ident_key},
 };
 use sway_core::{
-    declaration_engine,
+    declaration_engine::{self, de_get_struct},
```

Comment:
Change `declaration_engine::{self, de_get_struct},` to `declaration_engine`, and change references to `de_get_struct` into `declaration_engine::de_get_struct`